### ISSUE TITLE
Arguments in function can be placed on new line.

### ISSUE THREAD

**Issue by o01eg** [phildawes/racer#412]
Some functions can have arguments placed on different lines so racer returns such output:

```
PREFIX 12,12,
MATCH toml,18,4,/home/o01eg/.cargo/registry/src/github.com-0a35038f75765ae4/toml-0.1.22/src/decoder/mod.rs,StructField,toml
MATCH cur_field,19,4,/home/o01eg/.cargo/registry/src/github.com-0a35038f75765ae4/toml-0.1.22/src/decoder/mod.rs,StructField,cur_field
MATCH sub_decoder,111,7,/home/o01eg/.cargo/registry/src/github.com-0a35038f75765ae4/toml-0.1.22/src/decoder/mod.rs,Function,fn sub_decoder(&self, toml: Option<Value>, field: &str) -> Decoder
MATCH err,125,7,/home/o01eg/.cargo/registry/src/github.com-0a35038f75765ae4/toml-0.1.22/src/decoder/mod.rs,Function,fn err(&self, kind: DecodeErrorKind) -> DecodeError
MATCH mismatch,132,7,/home/o01eg/.cargo/registry/src/github.com-0a35038f75765ae4/toml-0.1.22/src/decoder/mod.rs,Function,fn mismatch(&self, expected: &'static str,
                found: &Option<Value>) -> DecodeErro
END
```

In the vim plugin which chooses lines by MATCH it cause incomplete list of the arguments.

**Comment by bruno-medeiros** [comment]
Yeah, this caused some issues for RustDT too.

**Comment by phildawes** [comment]
Am currently working on using rustc for perfect completions, so I'm not likely to work on this super soon. Somebody else might pick it up though

**Comment by andjo403** [comment]
I think this can be closed shall be fixed in #585

**Comment by jwilm** [comment]
Thanks for mentioning it @andjo403; this can definitely be closed (already resolved, maybe pending release).
